### PR TITLE
Version bump & code clean up.

### PR DESCRIPTION
**Pull request by GATHONI-WANJIRA**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,10 +3,9 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
     <groupId>org.apluscreators</groupId>
     <artifactId>Google-Translate-Labs</artifactId>
-    <version>1.0-SNAPSHOT</version>
```
Comment (samuelowino) [inline pom.xml:9]: This should move to 2.0-SNAPSHOT not BETA

**Review by samuelowino — CHANGES_REQUESTED** [review]
Fix these changes for the code be merged

File: pom.xml (hunk 1 above)
Comment (GATHONI-WANJIRA) [reply pom.xml:9]: Noted. 

Thank you.

**Comment by samuelowino** [comment]
Wonderful, this looks ready.

Great work

**Pull request merged**